otter/tap/api.py

```diff
@@ -278,17 +279,22 @@ def on_client_ready(_):
                                              get_service_configs(config),
                                              kz_client, store, supervisor,
                                              cassandra_cluster)
+
             # Setup scheduler service after starting
             scheduler = setup_scheduler(parent, dispatcher, store, kz_client)
-            health_checker.checks['scheduler'] = scheduler.health_check
-            otter.scheduler = scheduler
+            if scheduler is not None:
```

Comment:
Thanks.

Active Nothing is more popularly known as Null Object https://en.wikipedia.org/wiki/Null_Object_pattern but I like Active Nothing because Null Object sounds more like it might be just None :).

In this case it would mean that `setup_scheduler` would define a type of a given interface, and it would never return `None`, but it might return one of two different implementations of that interface, one of which does nothing.